File: go/minimum-number-of-steps-to-make-two-strings-anagram.go
func minSteps(s string, t string) int {
    offset := int('a')
    var charCount [26]int
    steps := 0
    for i := 0; i < len(s); i++ {
        s_j := int(s[i]) - offset
        charCount[s_j]++
        t_j := int(t[i]) - offset
        charCount[t_j]--
    }
    for i := 0; i < 26; i++ {
        if 0 < charCount[i] {
            steps += charCount[i]
        }
    }
    return steps
}

File: python/minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        offset = ord('a')
        char_count = [0] * 26
        min_steps = 0
        for i in range(len(s)):
            s_j = ord(s[i]) - offset
            char_count[s_j] += 1
            t_j = ord(t[i]) - offset
            char_count[t_j] -= 1
        for c in char_count:
            if 0 < c:
                min_steps += c
        return min_steps
